test/integration/generated_logs_test.go

```diff
@@ -0,0 +1,173 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+// Verifies logs generated in the /tmp directory
+func TestGeneratedLogs(t *testing.T) {
```

Comment:
I am not keen on making a new cluster just for this test, unless there was a "blocking" reason not to test this with TestErrorSpam ?

Reply:
Moved the changes to the previous file